.travis.yml

```diff
@@ -254,7 +255,7 @@ script:
         - make -j $(nproc)
         - mkdir /dev/shm/odp
         - if [ -z "$CROSS_ARCH" ] ; then
-          sudo LD_LIBRARY_PATH="$HOME/cunit-install/$CROSS_ARCH/lib:$LD_LIBRARY_PATH" ODP_SHM_DIR=/dev/shm/odp make check ;
+          sudo ODP_CONFIG_FILE="`pwd`/config/odp-linux.conf" LD_LIBRARY_PATH="$HOME/cunit-install/$CROSS_ARCH/lib:$LD_LIBRARY_PATH" ODP_SHM_DIR=/dev/shm/odp make check ;
```

Comment:
OFP is using a similar naming for the environment variable but with 'CONF' instead of 'CONFIG' ('OFP_CONF_FILE'). Will it makes sense to change the naming to avoid confusions: either 'ODP_CONF_FILE' in ODP or 'OFP_CONFIG_FILE' in OFP?

Reply:
I would prefer changing OFP to match our naming convention. We use 'config' in several APIs, whereas 'conf' is used only internally.